.github/workflows/config_docs.yml

```diff
@@ -62,8 +62,8 @@ jobs:
       - name: Checkout docs website repository
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # tag v4.2.2
         with:
-          repository: ${{ env.DESTINATION_REPO }}
           token: ${{ secrets.NEWRELIC_RUBY_AGENT_BOT_TOKEN }}
+          repository: ${{ env.DESTINATION_REPO }}
```

Comment:
This was an accident when reverting some other changes; lmk if you prefer I undo this! (There's a similar one below too)